build/project-template-gradle/build.gradle

```diff
@@ -270,6 +270,17 @@ copyMetadata.dependsOn(buildMetadata)
 
 task buildapk {
 	
+	  task includePluginBuildFiles {
+
+      new File("$nodeModulesDir").eachFileRecurse(groovy.io.FileType.FILES) {
```

Comment:
It is already planned and discussed with @Fatme. There is an issue about that as well, but couldn't find it. 

This pull request will be replaced by more broader functionality which includes CLI logic copying the include.gradle into a special folder for every plugin on prepare step instead of on plugin add (since plugins can be added before platform add). This will remove the recursive search for this file. All this will happen in 1.5. For now this pull is "good enough", ie it may never get into release in its current form.